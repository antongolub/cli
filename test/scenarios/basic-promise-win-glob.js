'use strict'
const requireInject = require('require-inject')
const mockProcess = Object.assign({}, process, {
  platform: 'win32',
  argv: [process.argv[0], process.argv[1], 'basic-promise-win-gl*.js'],
  cwd: () => __dirname
})
requireInject('../..', {process: mockProcess})(main)

function main (_, args) {
  console.log('started', args)
  return new Promise((resolve, reject) => {
    console.log('finished')
    resolve()
  })
}
