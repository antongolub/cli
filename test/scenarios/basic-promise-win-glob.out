started basic-promise-win-glob.js
finished
0
